ssg/constants.py

```diff
@@ -516,6 +522,7 @@
     'example': 'Example Linux Content',
     'ol': 'Oracle Linux',
     'ocp': 'Red Hat OpenShift Container Platform',
+    'rhcos': 'Red Hat Enterprise Linux CoreOS',
```

Comment:
When this line is like this:
`'rhcos': 'Red Hat Enterprise Linux CoreOS release'`,

The build process includes OVALs included by `affected_platform`, like `auditd_data_retention_flush`.

What happens is that in `is_applicable_for_product()` https://github.com/ComplianceAsCode/content/blob/master/ssg/utils.py#L123, it tries to reconstruct the full name of the product, which in rhcos4/product.yml is defined with trailing `release`.

Now I wonder how much the word `release` is actually really part of the name.
Should the product full_name in rhcos4/product.yml be `Red Hat Enterprise Linux CoreOS 4`?

Reply:
let's give it a try